src/Symfony/Component/Console/Attribute/AsLockedCommand.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Attribute;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class AsLockedCommand extends AsCommand
```

Comment:
Using a separate attribute (not extending `AsCommand`) would be more appropriate to compose with other behaviors if there is.

Reply:
Yes, I wasn't sure about the way to do it.

Maybe just add `lock` and `blocking` properties in the `AsCommand` attribute would be even better ?